src/main/scala/uk/gov/nationalarchives/draftmetadatavalidator/LambdaRunner.scala

```diff
@@ -0,0 +1,18 @@
+package uk.gov.nationalarchives.draftmetadatavalidator
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream}
+
+object LambdaRunner extends App {
+  private val body =
+    """{
+      |  "consignmentId": "f82af3bf-b742-454c-9771-bfd6c5eae749",
+      |  "fileName": "TDR-2024.csv"
```

Comment:
Does the input have a `fileName`? Though its just a consignment id at the moment?